### ISSUE TITLE
Ordenar por custo benefício do investimento

### ISSUE THREAD

**Issue by marcodotcastro** [desenvolvendome/growth_channel#32]
Como youtuber, gostaria de:

 - [ ] visualizar todos os vídeos em uma lista ordenada pelos maiores custos benefício

Informações geradas através do CSV exportado da plataforma do Youtube para analisar o desempenho.

**Comment by idylicaro** [comment]
opa @marcodotcastro não consegui visualizar essa tag no csv, ela é realmente uma tag ?

**Comment by marcodotcastro** [comment]
> opa @marcodotcastro não consegui visualizar essa tag no csv, ela é realmente uma tag ?

Não. Existem uma regra para análise de custo benefício no sistema.
